src/Core/Cache/Capability/ICanCacheInMemory.php

```diff
@@ -53,4 +53,11 @@ public function compareSet(string $key, $oldValue, $newValue, int $ttl = Duratio
 	 * @throws CachePersistenceException In case the underlying cache driver has errors during persistence
 	 */
 	public function compareDelete(string $key, $value): bool;
+
+	/**
+	 * Returns some basic statistics of the used Cache instance
+	 *
+	 * @return array Returns an associative array of statistics
+	 */
+	public function getStats(): array;
```

Comment:
Adding a new method in an interface is a breaking change. Is this ok?

Reply:
Since it's only used by the given classes, which I already updated - think so :-)